spec/tree_spec.rb

```diff
@@ -1,14 +1,156 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
```

Comment:
this error here is mine. i've mistakenly written this as a string instead of referring to the `Tree` object.